### DISCUSSION TITLE
Trying report a spam account but "Missing expected parameters"

### DISCUSSION THREAD

**Discussion by BdR76** [community/community#35315] [Code Security]
I was trying to report an account that is [spamming the comment section](https://gist.github.com/4048722#gistcomment-4325723), but when I fill out the form here

- I want to report abusive content or behavior >
- I want to report SPAM, a user that.. >
- A user's profile or repository is spam >
- form below

I fill in a text message about the issue, but when I submit I get an error page:

> Sorry, we had a problem creating your request. Please correct the errors below and try again.

> 
> Missing expected parameters. So that we can accurately identify the user or content you wish to report please return to [GitHub](https://github.com/) and follow the instructions in the [Reporting Abuse or Spam](https://docs.github.com/github/building-a-strong-community/reporting-abuse-or-spam) article.
![github_cant_send_spam_report](https://user-images.githubusercontent.com/7793956/194280048-c88c3af5-fc96-423e-b32d-f0ef76d302e8.png)

**Comment by soulspark666** [comment]
Did you find any solution, having a similar issue at my end?

**Reply by BdR76** [reply]
No, the "report Github user" just seems to be broken. I ended up using another way to report it, that user has been removed now, but I can't remember how I did it or where the link was.

It might have been reporting on the actual comment itself not on the user, but I'm not sure. I can re-find it now. The github feedback forms seem a bit like a maze tbh.
